docs/security/certificate-management.md

```diff
@@ -744,9 +245,9 @@ NOTE:
     Run `osg-ca-manage refreshCA` and `osg-ca-manage fetchCRL` to commit your changes immediately.
 ```
 
-#### Removing/Adding a particular CA included in OSG CA package ####
+#### Removing/Adding a particular CA included in OSG CA package
 
-##### Removing #####
+##### Removing
```

Comment:
Similarly, combine these two into an h4, `Removing a CA from the OSG CA package`